### PR TITLE
update copyright year + suggested markup for copyright

### PR DESCRIPTION
**Pull request by deniak**

### CODE REVIEW

File: lib/rules.json [hunk 1]
```diff
@@ -799,7 +799,7 @@
...
.../Consortium/Legal/copyright-documents\">document use</a> rules apply.</p> </div> <div class=\"source\"> <span style=\"font-style: italic\">Include this source code</span>:<br><code>&lt;p class=\"copyright\"&gt;&lt;a href=\"https://www.w3.org/Consortium/Legal/ipr-notice#Copyright\"&gt;Copyright&lt;/a&gt; © 2017 &lt;a href=\"https://www.w3.org/\"&gt;&lt;abbr title=\"World Wide Web Consortium\"&gt;W3C&lt;/abbr&gt;&lt;/a&gt;&lt;sup&gt;®&lt;/sup&gt; (&lt;a href=\"https://www.csail.mit.edu/\"&gt;&lt;abbr title=\"Massachusetts Institute of Technology\"&gt;MIT&lt;/abbr&gt;&lt;/a&gt;, &lt;a href=\"https://www.ercim.eu/\"&gt;&lt;abbr title=\"European Research Consortium for Informatics and Mathematics\"&gt;ERCIM&lt;/abbr&gt;&lt;/a&gt;, &lt;a href=\"https://www.keio.ac.jp/\"&gt;Keio&lt;/a&gt;, &lt;a href=\"http://ev.buaa.edu.cn/\"&gt;Beihang&lt;/a&gt;). W3C &lt;a href=\"https://www.w3.org/Consortium/Legal/ipr-notice#Legal_Disclaimer\"&gt;liability&lt;/a&gt;, &lt;a href=\"https://www.w3.org/Consortium/Legal/ipr-notice#W3C_Trademarks\"&gt;trademark&lt;/a&gt; and &lt;a href=\"https://www.w3.org/Consortium/Legal/copyright-documents\"&gt;document use&lt;/a&gt; rules apply.&lt;/p&gt;</code></div>",
```
Comment (tripu) [inline lib/rules.json]: `The <code>abbr</code> element <span class=\"rfc2119\">MAY</span> be used in place of <code>abbr</code>.`

The whole sentence can be deleted now. (It's above in a few other cases, too.)

File: lib/rules.json [hunk 2]
```diff
@@ -34,7 +34,7 @@
...
```
Comment (tripu) [inline lib/rules.json]: There's still one occurrence of `2016` here, instead of `2017`. (It's below in a few other cases, too.)

**Comment by deniak** [comment]
@tripu Thanks. I addressed your 2 comments.
Regarding the regex you are mentioning, it shouldn't be updated. It's there just to say starting from August 2016, we are forcing `https` links for the previous version.

**Comment by tripu** [comment]
Holding it until the change of the year.

**Pull request merged**